src/main/java/ru/itis/controllers/WishListController.java

```diff
@@ -28,7 +28,7 @@ public WishListController(ItemService itemService, WishListService wishListServi
     @GetMapping("/lists/{list-id}")
     @PreAuthorize("isAuthenticated()")
     @ApiOperation("Get a single wish list")
-    public WishListDto getWishList(@PathVariable Long listId) {
+    public WishListDto getWishList(@PathVariable("list-id") Long listId, @RequestHeader(name = "AUTH") String token) {
```

Comment:
Do we need this unused parameter `@RequestHeader(name = "AUTH") String token`?

Reply:
If we allow only authorized users to enter this page, then yes